### PR TITLE
refactor(OnyxInput): use new matrix screenshot utility

### PR DESCRIPTION
**Pull request by larsrickert**
Relates to #871

### CODE REVIEW

File: packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx [hunk 1]
```diff
@@ -1,7 +1,88 @@
-import { executeScreenshotsForAllStates } from "../../utils/playwright";
+import { DENSITIES } from "../../composables/density";
 import { expect, test } from "../../playwright-axe";
+import { executeMatrixScreenshotTest } from "../../playwright/screenshots";
 import OnyxInput from "./OnyxInput.vue";
 
+test.describe("Screenshot tests", () => {
+  for (const state of ["default", "placeholder", "with-value", "autofill"] as const) {
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx:7]: might be prettier on the screenies?
```suggestion
  for (const state of ["default", "placeholder", "with value", "autofill"] as const) {
```
(remember to adjust line 18 accordingly)

File: packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx [hunk 2]
```diff
@@ -1,7 +1,88 @@
...
+        if (row === "focus") await input.focus();
+        if (state == "autofill") {
+          await input.evaluate((node) => node.setAttribute("data-test-autofill", ""));
+        }
+      },
+    });
+  }
+
+  executeMatrixScreenshotTest({
+    name: "Input (other)",
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx:35]:
```suggestion
    name: "Input (content around)",
```
or
```suggestion
    name: "Input (frame)",
```

File: packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx [hunk 3]
```diff
@@ -59,55 +140,6 @@ test("should emit events", async ({ mount, makeAxeBuilder }) => {
...
-
-  // ASSERT
-  await expect(component).toHaveScreenshot("required.png");
-});
-
-test("should show optional marker", async ({ mount }) => {
-  // ARRANGE
-  const component = await mount(
-    <OnyxInput
-      label="Very long label that should be truncated"
```
Comment (BoppLi) [inline packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx:74]: we have no truncation example anymore, or did I miss it?

**Review by BoppLi — CHANGES_REQUESTED** [review]

File: packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx (hunk 3 above)
Comment (BoppLi) [reply packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx:74]: maybe combine that with the long label with required + optional
and you could also pass in a very long placeholder / filled text.

Comment (larsrickert) [reply packages/sit-onyx/src/components/OnyxInput/OnyxInput.ct.tsx:74]: Good catch, I added it and also for the invalid state

**Review by BoppLi — APPROVED** [review]

**Pull request merged**